README.md

```diff
@@ -56,19 +56,18 @@ within Juttle. To do so, use the following format when adding a Postgres configu
 ```json
 {
     "adapters": {
-        "postgres": {
-            "connection": "postgres://postgres@localhost:5432/postgres"
-        }
+        "postgres": [{
```

Comment:
Yes but:
1. This `instances` stuff is not implemented yet in the juttle repo. Should I do that? @demmer?
2. I'll change the readme to the correct format when I bump to a juttle version that uses `instances`.